CHANGELOG.md

```diff
@@ -10,11 +10,44 @@ We appreciate your patience while we speedily work towards a stable release of t
 
 <!-- NEW CONTENT GENERATED BELOW. PLEASE PRESERVE THIS COMMENT. -->
 
+### 0.61.75 (2024-03-19)
```

Comment:
I think this should be v0.61.76; looks like .75 was the test fix

Reply:
oof sorry, good catch